Assets/Plugins/UniRx/Scripts/Disposables/CancellationDisposable.cs

```diff
@@ -51,7 +53,15 @@ public CancellationToken Token
         /// </summary>
         public void Dispose()
         {
-            _cts.Cancel();
+            if (!IsDisposed)
+            {
+                _cts.Cancel();
+            }
```

Comment:
The `Dispose()` method is designed to be idempotent, allowing for multiple invocations. If a `CancellationTokenSource` object is created inside `CancellationDisposable` and gets disposed during the initial call to `Dispose()`, subsequent attempts to cancel it will result in an exception.